### ISSUE TITLE
Feature request: Different cleaning power settings for different zones.

### ISSUE THREAD

**Issue by MadJoker0815** [rand256/valetudo#30]
Hi,
first I want to thank for all the work you and Hypfer have done.
I love the function to run several zones at once.
It would be helpful to set the vacuum strength for specific zones.
Example set the "Kitchen" to "Turbo" and then the "Living Room" to "Balanced". This could be integrated in the zone setup.
Also for normal use the zone power setting should not affect the regular power setting.

Maybe this will be possible by adding another line in the zone information in /mnt/data/valetudo/config.json
This could just be the speed information like 60 for Balanced.

**Comment by mvmaastricht** [comment]
Yes! This I would LOVE too! For example to do a MAX (100%) cleaning of the mess our cats make in the area around their litter box!
